integration_tests/dbt_project.yml

```diff
@@ -97,3 +108,9 @@ seeds:
     engagement_note_data:
       +column_types:
         engagement_id: "{{ 'int64' if target.name == 'bigquery' else 'bigint' }}"
+    ticket_company_data:
```

Comment:
do we need to add these if we're not configuring any column types or anything?

Reply:
We most likely don't. They were carried over in case I needed to do any column config for seed data that I never ended up using. I can remove!